components/CasesSummary.vue

```diff
@@ -20,17 +20,17 @@
         </div>
       </div>
       <div class="case-frequency">
-        <span class="stat-title">Confirmed Cases per 10,000</span>
+        <span class="stat-title">Confirmed Cases per 100,000</span>
         <div class="stat-number">
           {{
             (
-              calcTotalCasesSummary.cases /
-              (calcTotalCasesSummary.population / 10000)
+              calcAverageCasesInLast14Days /
+              (calcTotalCasesSummary.population / 100000)
             ).toFixed(2)
           }}
         </div>
         <div class="stat-note">
-          in past 14 days
+          in past 14 days average
```

Comment:
it might sound better if it was "on average in the past 14 days"

Reply:
Good catch! Fixed it.